vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -69,23 +76,40 @@ public void waitForElementDisappearance(Locator locator)
     }
 
     /**
-     * Checks that element located by <b>locator</b> exists during the <b>duration</b>
-     * @param locator locator to find an element
-     * @param duration total waiting time according to <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a>
-     * standard
+     * Validates the element located by the locator has existed for the period specified by the duration.
+     * The actions performed by the step:
+     * <ul>
+     * <li>check the search context is set,</li>
+     * <li>search for the element to validate existence (this search may include wait for element appearance if it's
+     * configured),</li>
+     * <li>validate the element has presented for the period specified by the duration.</li>
+     * </ul>
+     *
+     * @param locator  The locator to find an element.
+     * @param duration The period of time the element should have existed in
+     *                 <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a>
+     *                 format.
      */
     @Then("element located `$locator` exists for `$duration` duration")
     public void doesElementByLocatorExistsForDuration(Locator locator, Duration duration)
     {
-        String prettyDuration = formatDuration(duration);
+        String prettyPrintedDuration = formatDuration(duration);
```

Comment:
inline?

Reply:
we want to fail fast in case invalid duration (see the case from the unit tests)